model/jpa/src/main/resources/META-INF/jpa-changelog-2.2.0.xml

```diff
@@ -57,5 +57,13 @@
 
     </changeSet>
 
+    <changeSet author="keycloak" id="2.2.0.1">
```

Comment:
Please don't introduce extra changeSet for a specific version, we only want one changeset per-version. Just append to existing changeSet element if there is one.